features/issues/components/issue-list/issue-list.tsx

```diff
@@ -62,16 +67,55 @@ const PageNumber = styled.span`
   ${textFont("sm", "medium")}
 `;
 
+const FilterContainer = styled.div`
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  justify-content: flex-end;
+  gap: 1rem;
+  margin-bottom: 25px;
+  @media (max-width: ${breakpoint("desktop")}) {
+    & div {
+      flex: 1;
+    }
+  }
+`;
+
+function getParsedInfo(
+  value: string | undefined,
+  enumToMatch: object
+): string | undefined {
+  if (value === undefined) return undefined;
+  const enumKeys = Object.keys(enumToMatch);
+  if (enumKeys.includes(value))
+    return enumToMatch[value as keyof typeof enumToMatch];
+  return undefined;
+}
+
 export function IssueList() {
   const router = useRouter();
+
   const page = Number(router.query.page || 1);
+
+  const level = getParsedInfo(
+    router.query.level as string | undefined,
+    LevelEnum
+  );
+  const status = getParsedInfo(
+    router.query.status as string | undefined,
+    StatusEnum
+  );
```

Comment:
I'm curious why you couldn't do something like `const status = StatusEnum[router.query.status]`. Was that a TS issue?

Also the LevelEnum is already defined in `api/issue.types.ts` 😉

Reply:
Ok I see `getParsedInfo` also validates that the filter value is one of the accepted values defined in the enum. Makes sense